docs/resources/post_process_aws_connection.md

```diff
@@ -0,0 +1,60 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "clumio_post_process_aws_connection Resource - terraform-provider-clumio"
+subcategory: ""
+description: |-
+  Post process Clumio AWS Connection Resource used to post-process AWS connection to Clumio.
+---
+
+# clumio_post_process_aws_connection (Resource)
+
+Post process Clumio AWS Connection Resource used to post-process AWS connection to Clumio.
+
+## Example Usage
+
+```terraform
+resource "clumio_post_process_aws_connection" "example" {
+  # example configuration here
+  token                              = "mytoken"
+  role_external_id                   = "role_external_id"
+  account_id                         = "account_id"
+  region                             = "region"
+  role_arn                           = "role_arn"
+  config_version                     = "1"
+  discover_version                   = "3"
+  protect_config_version             = "18"
+  protect_ebs_version                = "19"
+  protect_rds_version                = "18"
+  protect_ec2_mssql_version          = "1"
+  protect_warm_tier_version          = "2"
+  protect_warm_tier_dynamodb_version = "2"
+  protect_s3_version                 = "1"
```

Comment:
I wonder if we should remove the "specific" versions from the example and rather go with the simplest example where if the versions are not provided (since they are optional) because the example looks complicated and also we may not always update this example each time there's a version change?

Reply:
This resource will not be used by the users directly. This will be a resource inside the module. Users will never invoke this directly.

Reply:
Got it, and just like the callback resource, the versions are required to be passed right?